test/boost/cdc_test.cc

```diff
@@ -369,7 +370,7 @@ SEASTAR_THREAD_TEST_CASE(test_cdc_log_schema) {
         assert_has_column(cdc::log_data_column_deleted_elements_name("c_typ"), set_type_impl::get_instance(short_type, false));
 
         // Check if we missed something
-        BOOST_REQUIRE_EQUAL(required_column_count, log_schema->all_columns_count());
+        BOOST_REQUIRE_GE(required_column_count, log_schema->all_columns_count());
```

Comment:
Why is this needed?
assert_has_column(cdc::log_meta_column_name("eor"), boolean_type); should have increased required_column_count by 1, no?

Reply:
It is not, but I found the check pernickety. Have _more_ columns than those required should not really be a problem.

Reply:
I think it's better to be precise. Otherwise we may miss errors when one column is added but tests are not updated and then another column is added and tests are just updated by +1. Then the test wouldn't catch the second column not working.

Reply:
Reverted